muxink/src/lib.rs

```diff
@@ -220,8 +222,10 @@ pub(crate) mod tests {
 
     // #[test]
     // fn from_bytestream_to_multiple_frames() {
-    //     let input = &b"\x06\x00\x00ABCDE\x06\x00\x00FGHIJ\x03\x00\xffKL\x10\x00\xffSINGLE_FRAGMENT\x02\x00\x00C\x02\x00\x00R\x02\x00\x00U\x02\x00\x00M\x02\x00\x00B\x02\x00\xffS"[..];
-    //     let expected: &[&[u8]] = &[b"ABCDEFGHIJKL", b"SINGLE_FRAGMENT", b"CRUMBS"];
+    //     let input =
```

Comment:
I don't think we need this change here, it'll just produce merge conflicts down the road.

Reply:
I reverted the change and added the `#[rustfmt::skip]` which should be removed after the tests are re-enable.

Updated in: a8056e4faf8b3846b6f9d3a2ba60b41e8682f9a9